models/user/user.go

```diff
@@ -301,18 +299,20 @@ func (u *UserStruct) CreateGithubUser(token string, githubUser *github.User, pri
 	u.UserName = sql.NullString{String: *githubUser.Login, Valid: true}
 	u.Uuid = sql.NullInt64{Int64: int64(*githubUser.ID), Valid: true}
 	u.Avatar = sql.NullString{String: *githubUser.AvatarURL, Valid: true}
-	if !u.Save() {
-		return false
+	if err := u.Save(); err != nil {
+		return err
 	}
-	return true
+	return nil
 }
 
-func (u *UserStruct) UpdateGithubUserInfo(token string, githubUser *github.User) bool {
+func (u *UserStruct) UpdateGithubUserInfo(token string, githubUser *github.User) error {
```

Comment:
__[golint]__
* [golint] exported method UserStruct.UpdateGithubUserInfo should have comment or be unexported